notebook/notebookapp.py

```diff
@@ -238,6 +238,20 @@ def init_handlers(self, settings):
         handlers.extend(load_handlers('services.nbconvert.handlers'))
         handlers.extend(load_handlers('services.kernelspecs.handlers'))
         handlers.extend(load_handlers('services.security.handlers'))
+        
+        # BEGIN HARDCODED WIDGETS HACK
+        try:
+            import ipywidgets
+            handlers.append(
+                (r"/nbextensions/widgets/(.*)", FileFindHandler, {
+                    'path': ipywidgets.find_static_assets(),
+                    'no_cache_paths': ['/'], # don't cache anything in nbextensions
```

Comment:
It probably is, but is it worth the annoyance?  I have no idea what kind of performance increase it'll give.  Does the browser cache work regardless?